### ISSUE TITLE
Data appears to be misaligned

### ISSUE THREAD

**Issue by Kallen113** [Kallen113/Python-Data-Analysis-Ebay-Data-Web-Crawler-and-ETL#1]
Part of the data wrangling/transformation process is causing the data to be misaligned. Testing has confirmed that price and other data are misaligned with the seller data (ie, seller names, ratings, etc.) by about 1 row offset.

**Comment by Kallen113** [comment]
The price, condition, auction_bids_count, total_price, and buy_it_now_or_auction are misaligned with the rest of the columns. Additional testing indicates that the misalignment occurs during the initial execution of the webcrawler, not during the subsequent data pipeline and data cleaning after the lists are transformed into a Pandas' DataFrame.

**Comment by Kallen113** [comment]
The issue appears to be that the 5 columns that are misaligned with the rest--namely: price, condition, auction_bids_count, total_price, and buy_it_now_or_auction--have an extra sort of extraneous element on the webpage, above any of the actual listings on each page of listings. Namely, the real listings data appears below the div with class of "srp-river-results clearfix".

**Comment by Kallen113** [comment]
Commit # 77c9c33 resolves this issue since the scraped data are now aligned properly. 
To achieve this, I needed to hone in on elements on the webpages that are specific to just the listing data themselves, and avoid the extraneous data lying just below the search bar, but above the first listing on any given page of listings. Namely, the listings data exists as child elements of a ul element with a class name of "class='srp-results srp-list clearfix". 

The aforementioned commit revised the script such that--instead of scraping the desired price data and other attribute elements via the entire webpage--it instead searches for and scrapes those elements from the confines of the ul element. 

However, to keep the data aligned properly, I needed to remove the columns related to bids counts and auction vs buy it now. The reason for this is--given the control flow of the web crawler script and the data structures I'm using--the script's scraped data are outputted to lists, and there is no way to readily match the number of bids with that of the other lists of scraped data. I would need to use different data structures or different control flow to ensure the bid and auction data matched up properly with that of the other scraped attributes. All other columns, however, have been retained and are now aligned properly.
